### PR TITLE
feat(Wolvesville): Game mode, logo

### PR DESCRIPTION
**Pull request by down-bad**
**Describe the changes in this pull request:**
- Show game mode
- Show player state
- Update logo
- Other minor additions, improvements and fixes

Please don't kill me, but in order to get the game mode that the player is playing we have to access one little value in `localStorage`. Although this is disallowed by the guidelines, the game mode is an important piece of information that I was trying to include since I started working on this presence, and there is no other way of getting it. I hope you understand and thanks in advance.

image

</details>

### CODE REVIEW

File: websites/W/Wolvesville/presence.ts [hunk 1]
```diff
@@ -3,9 +3,233 @@ const presence = new Presence({
...
+			return "unknown";
+	}
+}
+
 presence.on("UpdateData", async () => {
 	const privacyMode = await presence.getSetting<boolean>("privacy"),
 		privacyChat = await presence.getSetting<boolean>("privacyChat"),
+		gameLang = await presence.getSetting<boolean>("gameLang"),
 		showTimestamp = await presence.getSetting<boolean>("showTimestamp"),
 		logo = await presence.getSetting<number>("logo"),
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:234]: Use Promise.all

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/W/Wolvesville/presence.ts [hunk 2]
```diff
@@ -285,49 +566,133 @@ presence.on("UpdateData", async () => {
...
+					let gamemode: string,
+						playerState: string,
+						gameState = "",
+						gameOver = false,
+						rankedLeague = "unknown",
+						lang = "unknown";
+
+					if (!privacyMode) {
+						const gamemodeId = localStorage?.getItem("last-game-game-mode");
+						if (gamemodeId?.includes("sandbox")) gamemode = "Sandbox";
```
Comment (Bas950) [inline websites/W/Wolvesville/presence.ts:585]: @Slowlife01 @EncryptedDev we allow this or nah

Comment (EncryptedDev) [reply websites/W/Wolvesville/presence.ts:585]: Nah

Comment (EncryptedDev) [reply websites/W/Wolvesville/presence.ts:585]: A few presences do this if there is no other way of obtaining this data

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**